### ISSUE TITLE
[Feature] Link to DFU instructions on hub not found

### ISSUE THREAD

**Issue by laurensvalk** [pybricks/support#862]
**Is your feature request related to a problem? Please describe.**
On failed DFU connection, we show this:

![image](https://user-images.githubusercontent.com/12326241/208606024-c17a227e-fb25-4e62-989a-7f78981ab48b.png)

**Describe the solution you'd like**
Link to the instructions, just like we do in case of Bluetooth not found.

**Comment by laurensvalk** [comment]
I gave this a shot. Works, but not quite right. It leaves the notification open.

I tried modeling it after `NoHub.tsx` first, but 

`onAction('flashFirmware')` 

looked different than

 `dispatch(firmwareDfuWindowsDriverInstallDialogDialogShow())`.

Is this a different approach, or maybe I missed the equivalent form for this one?

![image](https://user-images.githubusercontent.com/12326241/208610597-e02af081-da69-48cc-8e25-4b9c56c80d15.png)
